hms-app/src/components/hackathon-details/CategoryDetails.tsx

```diff
@@ -72,27 +71,24 @@ export default function CategoryDetails(props: IProps) {
 
   useEffect(() => {
     loadSelectedCategory()
-    setCategoryData({
-      ...categoryData,
-      isLoadingCategoryData: true,
-    })
+    setIsLoading(true)
   }, [])
 
   return (
     <>
-      {categoryData.errorCategoryData && (
+      {isError && (
         <div>
           <h3>Error loading hackathons</h3>
           <p>something went wrong.</p>
         </div>
       )}
-      {categoryData.isLoadingCategoryData && (
+      {isLoading && (
         <div>
           <h3>Category details are loading...</h3>
         </div>
       )}
 
-      {!categoryData.isLoadingCategoryData && (
+      {!isLoading && (
```

Comment:
Maybe also :thinking: 
```suggestion
      {!isLoading && !isError (
```